otelcollector/build/windows/scripts/main.ps1

```diff
@@ -283,25 +349,6 @@ function Start-Telegraf {
         /opt/telegraf/telegraf.exe --service start
     }
 }
-
-function Start-ME {
-    Write-Output "Starting Metrics Extension"
-    Write-Output "ME_CONFIG_FILE = $env:ME_CONFIG_FILE"
-    Write-Output "AZMON_DEFAULT_METRIC_ACCOUNT_NAME = $env:AZMON_DEFAULT_METRIC_ACCOUNT_NAME"
-    Start-Job -ScriptBlock { 
-        $me_config_file = $env:ME_CONFIG_FILE
-        $AZMON_DEFAULT_METRIC_ACCOUNT_NAME = $env:AZMON_DEFAULT_METRIC_ACCOUNT_NAME
-        $ME_ADDITIONAL_FLAGS = $env:ME_ADDITIONAL_FLAGS
-        if (![string]::IsNullOrEmpty($ME_ADDITIONAL_FLAGS)) {
-            Start-Process -NoNewWindow -FilePath "/opt/metricextension/MetricsExtension/MetricsExtension.Native.exe" -ArgumentList @("-Logger", "File", "-LogLevel", "Info", "-DataDirectory", ".\", "-Input", "otlp_grpc_prom", "-MonitoringAccount", $AZMON_DEFAULT_METRIC_ACCOUNT_NAME, "-ConfigOverridesFilePath", $me_config_file, $ME_ADDITIONAL_FLAGS) > $null
-        }
-        else {
-            Start-Process -NoNewWindow -FilePath "/opt/metricextension/MetricsExtension/MetricsExtension.Native.exe" -ArgumentList @("-Logger", "File", "-LogLevel", "Info", "-DataDirectory", ".\", "-Input", "otlp_grpc_prom", "-MonitoringAccount", $AZMON_DEFAULT_METRIC_ACCOUNT_NAME, "-ConfigOverridesFilePath", $me_config_file) > $null
```

Comment:
Change logic in ME to work with MAC == false and true both (i.e. localcontrolchannel is not set)

Reply:
Tested Simple mode : Works as expected.

Reply:
Tested advanced mode : Works as expected (MAC == true)